### DISCUSSION TITLE
Ability to use query in Panel link

### DISCUSSION THREAD

**Discussion by PhilippHomann** [grafana/grafana#38251] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
I want to be able to use the query used in a panel as a URL variable.

**Why is this needed**:
I need the ability to export logs stored in Loki and displayed in a Log or Table panel.
My idea was to add a panel link, which points to a script, which will do the export and offer a file download.

Currently I have to pass all variables inside my dashboard and the time range manually to the script.
Also I have to maintain the query in two different locations (in my script and in the dashboard).
I'm looking for a way to pass the query used in the panel to my link.
Ideally the variables are already replaced by its values, so I just have to use the query parameter inside my script and pass it to logcli for example.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
